### PR TITLE
Two ali tuts

### PR DESCRIPTION
**Pull request by GreatGhostsss**
The description key for the Redis article is a little cheeky, but I think we can get away with it since the article title is descriptive. If you don't think it can work, let's try: 

> This post expands OKE with Terraform to new heights: Ali shows you how to configure OKE to interface with a Redis cluster while monitoring it with Prometheus, then visualizing with Grafana.

That maybe too long, though. LMK.

Otherwise, I got a message from JW to provide backlinks to my articles from last week, so I added some contents in those articles' front matter to visit dev.o.

### CODE REVIEW

File: deploying-monitoring-redis-oke.md [hunk 1]
```diff
@@ -0,0 +1,257 @@
+---
+title: Deploying and monitoring a Redis cluster to Oracle Container Engine (OKE)
+parent: [tutorials]
+thumbnail: assets/redis-dashboard.png
+tags: [backend, oci]
+date: 2021-12-15 13:34
+description: Redis, Prometheus, OKE, oh my!
+MRM: WWMK211213P00053
+author: ali-mukadam
```
Comment (ttscoff) [inline deploying-monitoring-redis-oke.md:9]: I think I may have given confusing instructions, but when the author exists in _authors.yml, the front matter key should be configured as `author: AUTHOR_SLUG`

File: deploying-monitoring-redis-oke.md [hunk 2]
```diff
@@ -0,0 +1,257 @@
...
+--set kubeControllerManager.enabled=false \
+--set kubeEtcd.enabled=false \
+--set kubeScheduler.enabled=false
+```
+
+Setting `serviceMonitorSelectorNilUsesHelmValues` to false ensures that all ServiceMonitors will be selected.
+
+Get a list of pods and identify the Prometheus pods:
+
+<!--EDIT there's a line break missing somewhere between grep and prom- but I'm not sure where-->
```
Comment (ttscoff) [inline deploying-monitoring-redis-oke.md:55]: You can remove my HTML comments after editing. Some of these commands appear to be mangled or poorly copy/pasted, please validate against whatever the original source material was.

File: deploying-monitoring-redis-oke.md [hunk 3]
```diff
@@ -0,0 +1,257 @@
...
+
+First, we have to generate the dataset. We will be using the mimesis package:
+
+```console
+pip install mimesis
+```
+
+And we will adapt the schema a little bit so we can make use of whatever mimesis provides to create a csv file using Python:
+
+<!--EDIT: This appears to be mangled python, please validate from source-->
```
Comment (ttscoff) [inline deploying-monitoring-redis-oke.md:161]: Please validate

File: deploying-monitoring-redis-oke.md [hunk 4]
```diff
@@ -0,0 +1,257 @@
...
+redis-master-0                   1/1     Running   0          156m                                                                               
+redis-metrics-794db76ff7-xmd2q   1/1     Running   0          156m                                                                               
+redis-slave-7fd8b55f7-25w8d      1/1     Running   1          156m                                                                               
+redis-slave-7fd8b55f7-hvhmc      1/1     Running   1          156m                                                                               
+redis-slave-7fd8b55f7-mjq8q      1/1     Running   1          156m
+```
+
+Afterwards, use port-forward to so you can access it using the redis-cli:
+
+<!--EDIT is there actually a command `k`?-->
```
Comment (ttscoff) [inline deploying-monitoring-redis-oke.md:194]: There might be a command `k`, but I'm assuming there's something missing. Please validate.

File: deploying-monitoring-redis-oke.md [hunk 5]
```diff
@@ -0,0 +1,257 @@
...
+
+Before we import the csv, access Grafana (http://localhost:3000) as described above by opening a third terminal and running kubectl port-forward locally. Browse to the Redis Dashboard and set the refresh to every 5 seconds:
+
+```console
+kubectl -n monitoring port-forward prom-operator-grafana-77cdf86d94-m8pv5 3000:3000
+```
+
+Now import the csv file as follows:
+
+<!--EDIT appears mangled, assume line break in --pipe\nAll, and no line break before file.csv-->
```
Comment (ttscoff) [inline deploying-monitoring-redis-oke.md:223]: I went ahead and modified this one so it made sense to me, but please double check it against source.

File: deploying-monitoring-redis-oke.md [hunk 6]
```diff
@@ -0,0 +1,257 @@
...
+
+You may want to watch the Redis dashboard in Grafana. You can see the immediate jump in Network IO, the number of items in the DB as well as the amount of memory used.
+
+{% imgx assets/redis-dashboard-after-insertion.png  "Redis Dashboard after mass insertion" %}
+
+While we installed the Prometheus Operator and Redis Cluster manually using the cli, you can also achieve that using the Terraform helm provider. As you are enabling monitoring on Redis, you need to ensure the relevant CRDs are created. When you are doing that manually and in the order above, this is done for you.
+
+However, when you use Terraform to do the provisioning, you will need to explicitly set the order as follows:
+
+<!--EDIT please verify indentation-->
```
Comment (ttscoff) [inline deploying-monitoring-redis-oke.md:241]: Seems like everything after the first line should be outdented one tab-width, but I defer to article's original formatting. Please verify.

**Review by ttscoff — CHANGES_REQUESTED** [review]
See comments inline.

File: deploying-monitoring-redis-oke.md (hunk 6 above)
Comment (GreatGhostsss) [reply deploying-monitoring-redis-oke.md:241]: For this one, I actually did match it to article's formatting (though I did add a space I missed btwn the first code block and the next line).

File: deploying-monitoring-redis-oke.md (hunk 5 above)
Comment (GreatGhostsss) [reply deploying-monitoring-redis-oke.md:223]: My new commit matches formatting form the source

File: deploying-monitoring-redis-oke.md (hunk 4 above)
Comment (GreatGhostsss) [reply deploying-monitoring-redis-oke.md:194]: Nah, this one is actually exactly faithful to the code block in the article, formatting and all. `k` isn't an option/flag as far as I can tell.

File: deploying-monitoring-redis-oke.md (hunk 3 above)
Comment (GreatGhostsss) [reply deploying-monitoring-redis-oke.md:161]: Corrected ✅

File: deploying-monitoring-redis-oke.md (hunk 2 above)
Comment (GreatGhostsss) [reply deploying-monitoring-redis-oke.md:55]: Corrected this and other goofily-formatted blocks

File: deploying-monitoring-redis-oke.md (hunk 1 above)
Comment (GreatGhostsss) [reply deploying-monitoring-redis-oke.md:9]: ✅

**Review by ttscoff — APPROVED** [review]

**Pull request merged**